sdk/dotnet/ApiExtensions/V1/Inputs/JSONSchemaPropsArgs.cs

```diff
@@ -100,7 +100,7 @@ public InputList<System.Text.Json.JsonElement> Enum
         public Input<string>? Id { get; set; }
 
         [Input("items")]
-        public InputUnion<Pulumi.Kubernetes.Types.Inputs.ApiExtensions.V1.JSONSchemaPropsArgs, ImmutableArray<System.Text.Json.JsonElement>>? Items { get; set; }
+        public InputUnion<Pulumi.Kubernetes.Types.Inputs.ApiExtensions.V1.JSONSchemaPropsArgs, ImmutableArray<InputJson>>? Items { get; set; }
```

Comment:
It's just two properties and I'm not sure how commonly used they are but I think this is technically breaking...  cc @pgavlin

Reply:
I _think_ that the new version is "more correct": it _should_ accept a wider set of values than the original definition, and I don't _think_ that user code that passes inputs to these properties will need to change. However, I'm not entirely certain there, and it is technically breaking from the POV of copying one of these fields to a variable that has an explicit type name. Do we have tests that exercise this field?

Reply:
Usually, an `Array<T>` won't convert to `Array<U>` implicitly even if `T` is implicitly convertable to `U`. So

```cs
// compiles
InputUnion<string, ImmutableArray<JsonElement>> works = new JsonElement[] { new JsonElement() }.ToImmutableArray();
// wont compile
InputUnion<string, ImmutableArray<InputJson>> fails = new JsonElement[] { new JsonElement() }.ToImmutableArray();
```

Reply:
Ugh, right

Reply:
Filed https://github.com/pulumi/pulumi/issues/7569

Reply:
@mikhailshilkov I pulled in @pgavlin's fix, so can you double-check that with the latest commit.

Reply:
The type has changed from `ImmutableArray` to `InputList` but I _think_ it's okay.

Reply:
Ah, damn--let me see if I can avoid that change. I'd rather it stick as `ImmutableArray` to minimize the diff.

Reply:
Updated my PR to fix that type diff.

Reply:
Pulled in @pgavlin's fix, so this should be resolved now.